.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "roles/galaxyprojectdotorg.galaxy-tools"]
 	path = roles/galaxyprojectdotorg.galaxy-tools
-	url = https://github.com/galaxyproject/ansible-galaxy-tools.git
+	url = https://github.com/mvdbeek/ansible-galaxy-tools.git
```

Comment:
Because we're automatically synchronizing this from upstream in a crontab, I'n afraid we may break that synchronization. The changes are already in https://github.com/galaxyproject/ansible-galaxy-tools/pull/31, so as soon as that get's merged in we will have it in the artbio fork as well. (Which I consider to be a backup as discussed in https://github.com/ARTbio/ansible-artimed/issues/62)